packages/helper-backend/src/langflow-mapping/dtos/create.dto.ts

```diff
@@ -12,3 +13,14 @@ export class CreateLangFlowMapping {
   @IsString()
   historyComponentID: string;
 }
+
+export class CreateLangFlowMappingDto {
+  @ApiProperty({ description: 'The model name' })
```

Comment:
I think you should be able to just add these annotations to the `CreateLangFlowMapping` class and remove this additional definition.